data/856/830/13/85683013.geojson

```diff
@@ -68,17 +74,21 @@
         "\u0395\u03c0\u03b1\u03c1\u03c7\u03af\u03b1 \u03a1\u03ac\u03c0\u03bb\u03b1"
     ],
     "name:eng_x_preferred":[
-        "Raplamaa"
```

Comment:
Should this be moved to English variant? (applies to others, too)? Perhaps we need to clarify lifecycle rules around name changes and localization?

Reply:
Especially since the default name is changing, and effectively English is default localization, but then the old default name is no longer listed in English variants either.

Reply:
Checked the lifecycle rule in this diagram, and we're good with your change as-is:

https://raw.githubusercontent.com/whosonfirst/whosonfirst-cookbook/master/_images/lifecycle_flowchart.png